R/pkg/NAMESPACE

```diff
@@ -3,7 +3,7 @@
 importFrom("methods", "setGeneric", "setMethod", "setOldClass")
 importFrom("methods", "is", "new", "signature", "show")
 importFrom("stats", "gaussian", "setNames")
-importFrom("utils", "download.file", "object.size", "packageVersion", "untar")
+importFrom("utils", "download.file", "object.size", "packageVersion", "tail", "untar")
```

Comment:
This was regressed from a recent commit. check-cran.sh actually is flagging this in an existing NOTE but we only check for # of NOTE (which is still 1), and so this went in undetected.